### ISSUE TITLE
Run R cells in vscode using jupyter

### ISSUE THREAD

**Issue by grst** [quarto-dev/quarto#204]
When executing a `{python}` cell in vscode it automatically starts a jupyter kernel and console. 
When executing a `{r}` cell, it instead relies on the vscode R extension to start a console. 
I would prefer R cells to be executed via jupyter as well, using an R kernel. 

This is mostly for two reasons:
 * The jupyter extension makes it really easy to work with different virtual environments. I can simply add a different kernel for each environment. I haven't found a way of specifying a virtual R environment in the vscode R extension except for manually setting paths in the configuration. 
 * the jupyter console looks and feels much nicer (even compared to `radian`) and I'm more used to it.

**Comment by jjallaire** [comment]
I agree that the Jupyter interactive window is much nicer! Currently however the API we use to do that doesn't allow us to specify a language (we just pass a string with code which is presumed to be Python) so can't execute R code in Jupyter. Definitely something we'd like to be able to do in the future though and we'll polk at it to see if there is another way.

**Comment by danieltomasz** [comment]
@grst I installed R jupyter kernel, and try to run R kernel in Jupyter notebookin Vscode, it works but I am not able to preview R variables via "Jupyter: Variables" did you found an workaround?

**Comment by grst** [comment]
May it be worth escalating this issue to vscode devs? Jupyter supports a ton of languages, so there's no good reason to assume it's Python.

**Comment by simonleandergrimm** [comment]
@danieltomasz Did you ever find a proper solution for this? I also chose an R kernel that enabled jupyter to render R. But for the R code to be sent to the interactive window, I still need to flag chunks with Python, which is kind of hacky:
![Image](https://github.com/user-attachments/assets/d6463d2f-5b92-4574-a6e9-e0c0dad7695b)
